saleor/order/utils.py

```diff
@@ -92,3 +92,25 @@ def cancel_order(order):
         cancel_delivery_group(group, cancel_order=False)
     order.status = OrderStatus.CANCELLED
     order.save()
+
+
+def add_item_to_delivery_group(group, variant, quantity):
```

Comment:
This function is very similar to `add_items_to_delivery_group`. It would be great if we could extract some common parts an reuse it.